src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -2,19 +2,22 @@
 
 import com.kauailabs.navx.frc.AHRS;
 import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.Servo;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.constants.Constants;
 import frc.robot.utils.smartshuffleboard.SmartShuffleboard;
 
 public class Climber extends SubsystemBase {
     private final CANSparkMax SparkMax1;
     private final CANSparkMax SparkMax2;
+    private final Servo servo;
     private final AHRS navxGyro;
 
-    public Climber(AHRS navxGyro){
+    public Climber(AHRS navxGyro, Servo servo) {
         this.SparkMax1 = new CANSparkMax(Constants.CLIMBER_MOTOR1_ID, CANSparkMax.MotorType.kBrushless);
         this.SparkMax2 = new CANSparkMax(Constants.CLIMBER_MOTOR2_ID, CANSparkMax.MotorType.kBrushless);
         this.navxGyro = navxGyro;
+        this.servo = servo;
```

Comment:
Just use the constants here instead of passing as a parameter (You'd need two of them)